src/main/scala/exercises/04_Options.scala

```diff
@@ -44,6 +62,9 @@ object Options {
    * to convert a possible numeric String (e.g. Some("12")) to an integer
    */
   def totalPeopleInRooms(rooms: Map[Int, Option[String]]): Int = {
-    error("Fix me")
+    rooms.values.foldRight(0)((b,a) => peopleInRoom(b) + a)
```

Comment:
Similarly, I'm not sure if foldLeft or foldRight is better. When I looked it up, I read (http://stackoverflow.com/questions/24370549/foldleft-v-foldright-does-it-matter) that the performance of the two is comparable because of how Scala implements their foldLeft and foldRight functions. I haven't seen a language like that before